utils/html.go

```diff
@@ -26,10 +26,7 @@ type HTMLTemplateWatcher struct {
 	stopped   chan struct{}
 }
 
-func NewHTMLTemplateWatcher(directory string) (*HTMLTemplateWatcher, error) {
-	templatesDir, _ := fileutils.FindDir(directory)
-	mlog.Debug("Parsing server templates", mlog.String("templates_directory", templatesDir))
-
+func NewHTMLTemplateWatcher(directories ...string) (*HTMLTemplateWatcher, error) {
```

Comment:
If we just generate the old files with the exact same path, we don't need anything changing here.

Reply:
Yup, done.